modules/installation-complete-user-infra.adoc

```diff
@@ -40,36 +40,36 @@ $ watch -n5 oc get clusteroperators
-cloud-credential                           4.6.0   True        False         False      29h
-cluster-autoscaler                         4.6.0   True        False         False      29h
-config-operator                            4.6.0   True        False         False      6h39m
-console                                    4.6.0   True        False         False      3h59m
-csi-snapshot-controller                    4.6.0   True        False         False      4h12m
-dns                                        4.6.0   True        False         False      4h15m
-etcd                                       4.6.0   True        False         False      29h
-image-registry                             4.6.0   True        False         False      3h59m
-ingress                                    4.6.0   True        False         False      4h30m
-insights                                   4.6.0   True        False         False      29h
-kube-apiserver                             4.6.0   True        False         False      29h
-kube-controller-manager                    4.6.0   True        False         False      29h
-kube-scheduler                             4.6.0   True        False         False      29h
-kube-storage-version-migrator              4.6.0   True        False         False      4h2m
-machine-api                                4.6.0   True        False         False      29h
-machine-approver                           4.6.0   True        False         False      6h34m
-machine-config                             4.6.0   True        False         False      3h56m
-marketplace                                4.6.0   True        False         False      4h2m
-monitoring                                 4.6.0   True        False         False      6h31m
-network                                    4.6.0   True        False         False      29h
-node-tuning                                4.6.0   True        False         False      4h30m
-openshift-apiserver                        4.6.0   True        False         False      3h56m
-openshift-controller-manager               4.6.0   True        False         False      4h36m
-openshift-samples                          4.6.0   True        False         False      4h30m
-operator-lifecycle-manager                 4.6.0   True        False         False      29h
-operator-lifecycle-manager-catalog         4.6.0   True        False         False      29h
-operator-lifecycle-manager-packageserver   4.6.0   True        False         False      3h59m
-service-ca                                 4.6.0   True        False         False      29h
-storage                                    4.6.0   True        False         False      4h30m
+authentication                             4.7.0   True        False         False      3h56m
+cloud-credential                           4.7.0   True        False         False      29h
```

Comment:
One of the IBM Z testers noticed that baremetal is missing in the list:
`baremetal                              	4.7.0   True    	False     	   False      2d20h`
Since "2d20h" probably has to be adapted to something that better fits to whats listed here. 
reference: https://docs.google.com/document/d/1oSxDfX1ZJPtUo4pZeHKerD_LgjuiuBGkE6qeeirHnMk/edit?disco=AAAALDztNt8

Reply:
@SNiemann15 Thank you! Adding that line in here.